src/contentScripts/other/selma/layout.ts

```diff
@@ -277,7 +277,10 @@ async function eventListener() {
 
   // Add Credit banner with toggle button
   const creditElm = await createCreditsBanner()
-  document.querySelector('.semesterChoice')!.appendChild(creditElm)
+  const table = document.querySelector('.semesterChoice') ?? document.querySelector('.pageContentHeader')
+  if (table != null) {
```

Comment:
`!=` should never be used as it doesn't compare strict inequality, allowing a type coercion to pass this inequality check. Always prefer `!==` and `===`.